lib/routes/account.js

```diff
@@ -502,8 +502,11 @@ module.exports = function (
           }
         }
 
-        function sendNewDeviceLoginNotification () {
-          if (!doSigninConfirmation && config.newLoginNotificationEnabled && requestHelper.wantsKeys(request)) {
+        function sendNewDeviceLoginNotification() {
+          // New device notification emails should only be sent for requesting keys and
+          // not performing a sign-in confirmation.
+          var shouldSendNewDeviceLoginEmail = config.newLoginNotificationEnabled && requestHelper.wantsKeys(request) && !doSigninConfirmation
```

Comment:
This looks like the logic is equivalent, only the order of the checks has changed and a comment added?